client/state/data-layer/wpcom/sites/blog-stickers/index.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * External Dependencies
+ */
+import { translate } from 'i18n-calypso';
+import { isArray } from 'lodash';
+
+/**
+ * Internal Dependencies
+ */
+import { SITES_BLOG_STICKER_LIST } from 'state/action-types';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { errorNotice } from 'state/notices/actions';
+import { addBlogStickerHandler } from 'state/data-layer/wpcom/sites/blog-stickers/add';
+import { mergeHandlers } from 'state/action-watchers/utils';
+
+export function requestBlogStickerList( { dispatch }, action ) {
+	dispatch(
+		http( {
+			method: 'GET',
+			path: `/sites/${ action.payload.blogId }/blog-stickers`,
+			body: {}, // have to have an empty body to make wpcom-http happy
+			apiVersion: '1.1',
+			onSuccess: action,
+			onFailure: action,
+		} ),
+	);
+}
+
+export function receiveBlogStickerList( store, action, next, response ) {
+	// validate that it worked
+	if ( ! response || ! isArray( response ) ) {
+		receiveBlogStickerListError( store, action, next );
+		return;
+	}
```

Comment:
I may be missing something, but shouldn't this func be doing something in the happy case? like redispatching ?